### ISSUE TITLE
Keyboard shortcuts and actions should be supported.

### ISSUE THREAD

**Issue by gspencergoog** [flutter/flutter#31946]
Currently, `RawKeyboardListener` can listen to keyboard events, and trigger callbacks based on those events.  However, on a hardware keyboard it isn't very easy to support standard keyboard shortcuts like TAB, ENTER, CTRL-Z, etc., since each control needs to be wrapped with a listener.

Flutter needs to have a shortcut mechanism which can map keys to actions, and then dispatch actions to focused widgets in order to execute them.

**Comment by gspencergoog** [comment]
Re-landed in https://github.com/flutter/flutter/pull/42278

**Comment by SahajRana** [comment]
Hi @gspencergoog, I have recently started working on flutter & it's been great using it with the help of devs like you guys. I'm wondering how can we use keyboard shortcuts for windows like a copy/paste, redo, undo in a text widget.

**Comment by gspencergoog** [comment]
@SahajRana, thanks for the interest.  At the moment, we're still working on implementing support for those things.  There is rudimentary support for cut/copy/paste on the `master` channel, and we're continuing to work on making it work as it should. For instance, text fields don't yet use the `Shortcuts` widget to implement the key bindings as they should, they still handle the raw keys themselves.

**Comment by SahajRana** [comment]
@gspencergoog, thanks for the response!. I will try using the `Shortcuts` widget as a workaround. I have one more issue for quite some time now, not sure should I open a new issue for it.

In the master channel versions (above `v1.9.0`), for Windows, three major things got changed. 
1. The `draggable` widget on drag is quite glitchy when moved a little fast. (& the overall desktop app seems slow).
2. The window UI scale has increased (I think the dpi ratio is increased from embedder).
3. The app window on resizing doesn't refresh it's layout until I take cursor inside the window box.

I initially thought it would be solved by time, but still on the latest version of flutter which I tried today `v1.10.15-pre.330`. The problem still persists due to which I have to revert back to `v1.9.0`. I hope you guys might help us out with this!
